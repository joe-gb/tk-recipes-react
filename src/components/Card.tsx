import React from "react";
import styled from "styled-components";

export const Card = styled.div`
  box-shadow: 0px 2px 6px 0px rgb(50 50 93 / 8%),
    0px 0px 0px 1px rgb(50 50 93 / 5%);
  position: relative;
  -webkit-transition: all 0.15s;
  transition: all 0.15s;
  background: #ffffff;
  border-radius: 8px;
  padding: 20px 40px;
  margin: 0 auto;
  max-width: 800px;
`;
