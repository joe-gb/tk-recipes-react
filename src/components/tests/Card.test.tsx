import React from "react";
import { render, screen } from "@testing-library/react";
import { Card } from "../Card";

test("Snapshot test", () => {
  let { container } = render(<Card>This is a card</Card>);
  expect(container).toMatchSnapshot();
  expect(container.textContent).toBe("This is a card");
});
