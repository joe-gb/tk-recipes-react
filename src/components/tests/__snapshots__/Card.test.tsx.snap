// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test 1`] = `
.c0 {
  box-shadow: 0px 2px 6px 0px rgb(50 50 93 / 8%), 0px 0px 0px 1px rgb(50 50 93 / 5%);
  position: relative;
  -webkit-transition: all 0.15s;
  -webkit-transition: all 0.15s;
  transition: all 0.15s;
  background: #ffffff;
  border-radius: 8px;
  padding: 20px 40px;
  margin: 0 auto;
  max-width: 800px;
}

<div>
  <div
    class="c0"
  >
    This is a card
  </div>
</div>
`;
